### PR TITLE
Separate subnet utils into a standalone package

### PR DESCRIPTION
**Pull request by errordeveloper**

### CODE REVIEW

**Comment by errordeveloper** [comment]
@justinsb could you please approve?

File: pkg/util/subnet/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,14 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["subnet.go"],
+    importpath = "k8s.io/kops/pkg/util/subnet",
+    visibility = ["//visibility:public"],
```
Comment (justinsb) [inline pkg/util/subnet/BUILD.bazel]: Cool - no dependencies :-)

**Comment by justinsb** [comment]
Yes, a nice clean refactor of self-contained functionality.  Thanks @errordeveloper 

/approve
/lgtm

**Pull request merged**